Packs/ProofpointServerProtection/.pack-ignore

```diff
@@ -1,2 +1,2 @@
 [file:README.md]
-ignore=RM102
\ No newline at end of file
+ignore=RM104
```

Comment:
you change it from rm102 to rm104, maybe you need both of them?

Reply:
need only 104